tutorials/clinical/10_ieeg_localize.py

```diff
@@ -124,8 +127,12 @@ def plot_overlay(image, compare, title, thresh=None):
 # This takes a while (~10 minutes) to execute so we skip actually running it
 # here::
 #
-#    reg_affine, _ = mne.transforms.compute_volume_registration(
-#        CT_orig, T1, pipeline='rigids', verbose=True)
+#    from dipy.align import center_of_mass, translation, rigid
+#    reg_data, reg_affine = dipy.align.affine_registration(
+#         moving=np.array(CT_orig.dataobj), moving_affine=CT_orig.affine,
+#         static=np.array(T1.dataobj), static_affine=T1.affine,
+#         pipeline=[center_of_mass, translation, rigid])
+#    CT_aligned = nib.MGHImage(reg_data.astype(np.float32), T1.affine)
```

Comment:
-1 on this, it means we have something wrong with our `compute_volume_registration` code that should be fixed. Our pipeline should do something equivalent to theirs. Maybe the simplest thing is to actually use their pipeline under the hood in our code?

Reply:
Yeah, we could do that

Reply:
Ah so the main thing we added that wasn't supported by their pipeline is `zooms`

Reply:
After thinking about it, I think this might be okay, the CT-MR alignment is incredibly finicky and does not refactor well with the much more stable MR-MR alignments. If you think it's worth looking into more detail what is going wrong, I'm fine but seems like not the best use of time to me. We have a solution that works and the API is pretty good. To me, it's much better that it works optimally and looks a little bit worse than if everything is all nicely factored but it's slightly inaccurate and I've spent a lot of time trying to fix this part already.

Reply:
>  ... the CT-MR alignment is incredibly finicky and does not refactor well with the much more stable MR-MR alignments... To me, it's much better that it works optimally and looks a little bit worse than if everything is all nicely factored but it's slightly inaccurate and I've spent a lot of time trying to fix this part already.

To me it sounds like you are implicitly arguing for getting rid of our own `compute_volume_registration` / `apply_volume_registration` code in favor of directly calling `dipy` functions. This is because the real use case we currently have for our own versions of these functions is exactly for doing MR-CT alignment. Should we get rid of them?

Reply:
Sounds reasonable to me

Reply:
@alexrockhill can you try reverting the resample-each-step in favor of `starting_affine`? Having recently made a PR to dipy I see that's all their code does really. If it indeed works as well as `affine_registration` (when all `niter` are used the same way) then I wouldn't mind keeping `compute_volume_registration`, because 1) it supports `zooms` and 2) stays in sync with our `compute_source_morph` code.

We can set `niter_*` to whatever they need to be to be equivalent with `affine_registration` because `compute_volume_registration` is a new function.

Reply:
This has not been addressed